tests/ci/setup.sh

```diff
@@ -61,8 +61,21 @@ if [[ "${ID}-${VERSION_ID}" == "centos-8" ]]; then
     cp -fv /etc/redhat-release "${TMPCI_DIR}"
     cp -fv tests/ci/files/rhel-8-4-0-os-release /etc/os-release
     cp -fv tests/ci/files/rhel-8-4-0-rh-release /etc/redhat-release
-    cp -fv /usr/share/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/
-    ln -sfv /etc/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/rhel-8.json
+    #cp -fv /usr/share/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/
+    #ln -sfv /etc/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/rhel-8.json
+
+    #TODO: Make sure REPOSITORY_PATH is initialized
+    echo "{
```

Comment:
@guy9050 please make the template a file

Reply:
@thrix do you mean to write the template into a file and then read it, or to pre-create that file?

Reply:
@guy9050 Yes, it was what we do for the other files (blueprint, ks, etc). Actually, I explain this at the comment above (about the workaround), check after the `TLDR:`.